src/test-reader/index.js

```diff
@@ -23,9 +23,10 @@ module.exports = class TestReader extends EventEmitter {
         const { paths, browsers, ignore, sets, grep } = options;
 
         const { fileExtensions } = this.#config.system;
-        const setCollection = await SetsBuilder.create(this.#config.sets, { defaultDir: require("../../package").name })
+        const envSets = env.parseCommaSeparatedValue(["TESTPLANE_SETS", "HERMIONE_SETS"]).value;
+        const setCollection = await SetsBuilder.create(this.#config.sets, { defaultPaths: ["testplane", "hermione"] })
```

Comment:
It used to be `defaultDir` with `package.json`-s name. Now it is straight up passed as an array with two places to find tests.

Not sure if anybody uses sets with unspecified `files` section though.